lib/requests/diaryFoods.js

```diff
@@ -11,8 +11,10 @@ const getMeals = () => {
     .then(meals => {
       meals.forEach(meal => populateMealTable(meal))
     })
-    .then(meals => calculateTotalCalories())
+    .then(() => calculateTotalCalories())
     .then(() => populateRemainingCalories())
+    .then(() => calculateDiaryTotalCaloriesConsumed())
```

Comment:
We might be able to group line 14 and 15 together into a parent function - and possibly do the same to line 16 and 17.